openmsupply/open-msupply
# Barcode data

Fixes #1522

# 👩🏻‍💻 What does this PR do? 
 
Creates the barcode table and syncs barcode data

# 🧪 How has/should this change been tested? 

- run the tests
- create a barcode on central and initialise the oms database; barcode should come across
- create a barcode on remote and check that it syncs back to central

## 💌 Any notes for the reviewer?

I've renamed the `[barcode]barcode` field to `[barcode]value` because I find the name duplication silly and also because diesel is sad if a column is the same as the table name. Thought that `[barcode]code` is logical but confusing given our other uses of the word 'code'.

## 📃 Documentation

_No user facing changes_
